custom_components/plugwise/binary_sensor.py

```diff
@@ -32,7 +32,7 @@ async def async_setup_entry(
         config_entry.entry_id
     ][COORDINATOR]
 
-    entities: list[PlugwiseBinarySensorEntity] = []
+    entities: list[BinarySensorEntity] = []
```

Comment:
This change is wrong, the previous type `PlugwiseBinarySensorEntity` is correct.